packages/core/src/WebViewDatadog/__utils__/getInjectedJavaScriptBeforeContentLoaded.ts

```diff
@@ -0,0 +1,26 @@
+import { formatAllowedHosts } from './formatAllowedHosts';
+
+export const DATADOG_MESSAGE_PREFIX = '[DATADOG]';
+
+export const getInjectedJavaScriptBeforeContentLoaded = (
+    allowedHosts?: string[],
+    injectedJavaScriptBeforeContentLoaded?: string
+): string =>
+    `
+    try{      
+      ${injectedJavaScriptBeforeContentLoaded}
+    }
+    catch (error) {
+      // do nothing
+    }
```

Comment:
We should execute our code first I believe, if the user executes some async code that takes a while to run here it will delay our code.
They might also fetch resources and we'd like to track those with Datadog as well.
Do you have any reason to run the user-defined code first?

Reply:
Good point